### DISCUSSION TITLE
Configure text output in client app

### DISCUSSION THREAD

**Discussion by svirs** [resend/react-email#1409] [Ideas]
### Goals

1. Viewing the text output of an email isn't configurable (htmlToTextOptions aren't available)

### Background

Without the ability to configure [htmlToTextOptions](https://github.com/resend/react-email/blob/canary/packages/render/src/options.ts#L3) on the text output which is called in the app here [here](https://github.com/resend/react-email/blob/74052e27688f4911bccc3ab7d9d8cbbdb09202ef/packages/react-email/src/actions/render-email-by-path.tsx#L37) we're stuck with defaults. (My issue is uppercasing which is affecting my handlebar template strings)

### Proposal

Unsure the best way to push these configs into the app, could have a local file that has these configs JSON file 🤷 Happy to assist with this work if there's interest and a preferred path forward!

**Comment by gabrielmfern** [comment]
Are you getting the text output out of the preview server to then be used? Generally users will go for either running `email export` or rendering it themselves then saving to files where they prefer. I don't think it's in the scope of the preview server to allow configurations on this.

**Reply by svirs** [reply]
I have been because it's fairly convenient but I totally understand why you'd expect more configuration through those methods and not the preview server!
